test/Debugger/MultipleContextStack.js

```diff
@@ -21,8 +21,8 @@ var script1 = WScript.LoadScript("\
   function Script1Func3() { scriptFunc3(); } \
   function setFunc2(func) { scriptFunc2 = func; } \
   function setFunc3(func) { scriptFunc3 = func; }",
-  "samethread", "Script1.js");

```

Comment:
Make sense however, for this particular test, it doesn't make any difference. Opening a GH issue to track this instead.